frontend/packages/console-app/src/components/quick-starts/catalog/QuickStartCatalog.tsx

```diff
@@ -1,45 +1,24 @@
 import * as React from 'react';
-import { Dispatch } from 'redux';
-import { connect } from 'react-redux';
 import { useTranslation } from 'react-i18next';
-import { RootState } from '@console/internal/redux';
 import { Gallery, GalleryItem } from '@patternfly/react-core';
 import { EmptyBox, LoadingBox } from '@console/internal/components/utils';
-import * as QuickStartActions from '../../../redux/actions/quick-start-actions';
-import {
-  getActiveQuickStartID,
-  getAllQuickStartStates,
-} from '../../../redux/reducers/quick-start-reducer';
-import { QuickStart, AllQuickStartStates } from '../utils/quick-start-types';
+import { QuickStart } from '../utils/quick-start-types';
 import { getQuickStartStatus } from '../utils/quick-start-utils';
 import QuickStartTile from './QuickStartTile';
 
 import './QuickStartCatalog.scss';
+import { QuickStartContext, QuickStartContextValues } from '../utils/quick-start-context';
```

Comment:
nit: sort import

Reply:
@sahil143 this one could still be improved